CRM/Core/Payment.php

```diff
@@ -1567,10 +1584,19 @@ public function subscriptionURL($entityID = NULL, $entity = NULL, $action = 'can
    * @return string
    */
   protected function getPaymentDescription($params, $length = 24) {
-    $parts = array('contactID', 'contributionID', 'description', 'billing_first_name', 'billing_last_name');
-    $validParts = array();
+    $parts = [
+      'contactID',
+      'contributionID',
+      'description',
+      'billing_first_name',
+      'billing_last_name',
+    ];
+    $validParts = [];
     if (isset($params['description'])) {
-      $uninformativeStrings = array(ts('Online Event Registration: '), ts('Online Contribution: '));
+      $uninformativeStrings = [
+        ts('Online Event Registration: '),
+        ts('Online Contribution: '),
```

Comment:
Not caused by this PR but I notice there's whitespace inside a `ts()` string here which is a no-no.